packages/typegpu/src/data/struct.ts

```diff
@@ -43,6 +43,8 @@ class TgpuStructImpl<TProps extends Record<string, AnyTgpuData>>
 
   public readonly byteAlignment: number;
   public readonly size: number;
+  isLoose = false as const;
+  isCustomAligned = false;
```

Comment:
Tiny nit, for consistency
```suggestion
  public readonly isLoose = false as const;
  public readonly isCustomAligned = false;
```